Fuchas/Libraries/security.lua

```diff
@@ -106,4 +106,11 @@ function lib.getPermissions(pid)
 	return copy
 end
 
+setmetatable(lib, {
+		__newindex = function()
+			error("the security lib is protected")
+		end,
+		__metatable = {}
+})
```

Comment:
i'll move all that chunk of code (same for `users`) into a `package.protect()` function

Reply:
sounds good

Reply:
> Please change what i said in comments and then i'll approve the pull request.

i might end up re-pr-ing, but will do